Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -300,7 +300,7 @@ extension ProtocolDeclSyntax: EntityNode {
     }
 
     func annotationMetadata(with annotation: String) -> AnnotationMetadata? {
-        return leadingTrivia.annotationMetadata(with: annotation)
+        return leadingTrivia.annotationMetadata(with: annotation) ?? protocolKeyword.leadingTrivia.annotationMetadata(with: annotation)
```

Comment:
```suggestion
        let trivias = [
            leadingTrivia,
            protocolKeyword.leadingTrivia,
            modifiers.leadingTrivia
        ] + attributes.map(\.leadingTrivia)
        return trivias.lazy.compactMap { $0.annotationMetadata(with: annotation) }.first
```

I found this is not sufficient. Below is a failure pattern.

```swift
@MainActor
/// @mockable
public protocol P {
    func f() -> String
}
```

Reply:
Good catch! thank you.